RubeesEat.IntegrationTests/PageObjects/BillDetailsPageObject.cs

```diff
@@ -20,4 +20,10 @@ public EntryLineControlObject[] GetEntryLines()
                .Select(e => new EntryLineControlObject(this, e))
                .ToArray();
     }
+    
+    public EditBillPageObject ClickUpdate()
+    {
+        InvokeAction("editBillButton");
+        return CreatePageObject<EditBillPageObject>();
+    }
```

Comment:
This function should be ClickEdit.
I just now got confused and thought this was the Update button inside the EditBill page.